scripts/lib/api/processHtml.test.ts

```diff
@@ -247,6 +249,26 @@ test("replaceSourceLinksWithGitHub()", () => {
   );
 });
 
+test("convertRubricsToHeaders()", () => {
+  const doc = Doc.load(`<p class="rubric">Example</p>
+    <p class="rubric">Examples</p>
+    <p class="rubric">References</p>
+    <p class="rubric">Reference</p>
+    <p class="rubric">Attributes</p>
+    <p class="rubric">Methods</p>
+    <p class="rubric">Simple examples</p>
+    <p class="rubric">Example code</p>`);
+  convertRubricsToHeaders(doc.$, doc.$main);
+  doc.expectHtml(`<strong>Example</strong>
+    <strong>Examples</strong>
+    <strong>References</strong>
+    <h2>Reference</h2>
```

Comment:
This should be `<strong>` because it's "member-specific", similar to an example. Rather than signifying the start of a new section with its own "code members", this is only added info about the current member.